open-api/rest-catalog-open-api.yaml

```diff
@@ -1903,6 +1926,35 @@ components:
       schema:
         type: string
 
+    idempotency-key:
+      name: Idempotency-Key
+      in: header
+      required: false
+      schema:
+        type: string
+        format: uuid
+        minLength: 36
+        maxLength: 36
+        example: "550e8400-e29b-41d4-a716-446655440000"
+      description: |
+        Optional client-provided idempotency key for safe request retries.
+
+        When present, the server ensures no additional effects for requests that carry the same
+        Idempotency-Key within the same operation/resource scope. If a prior request with this key
+        has been finalized, the server returns an equivalent final response without re-running the
```

Comment:
"Equivalent" sounds too restrictive... something akin to JSON with the same logical structure, but different line-by-line format.

Can we be more specific and say something like: "The response body may reflect a more recent state of the catalog than what it was at the time the original change was committed.".... basically taking the text on line 1945 out of the parentheses. WDYT?